homeassistant/components/sensor/wunderground.py

```diff
@@ -627,22 +630,31 @@ def _get_attributes(rest):
                   'Latitude and longitude must exist together'): cv.longitude,
     vol.Required(CONF_MONITORED_CONDITIONS):
         vol.All(cv.ensure_list, vol.Length(min=1), [vol.In(SENSOR_TYPES)]),
+    vol.Optional(CONF_ENTITY_NAMESPACE,
```

Comment:
This is wrong. Entity namespace is implemented on the entity component level: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity_component.py#L138

Reply:
Yes, but we're generating our own entity ids here. So the namespace is never used in the entity component level - we have to do on our own here :/ (There might be another way though.)